src/NetAnalyzers/Microsoft.CodeAnalysis.NetAnalyzers.md

```diff
@@ -2676,7 +2676,7 @@ Applications available over HTTPS must use secure cookies.
 |CodeFix|False|
 ---
 
-## [CA5383](https://docs.microsoft.com/visualstudio/code-quality/ca5383): Ensure Use Secure Cookies In ASP.Net Core
+## [CA5383](https://docs.microsoft.com/visualstudio/code-quality/ca5383): Ensure Use Secure Cookies In ASP.NET Core
```

Comment:
These changes were automated when I had proper_names from dotnet/docs config. I think parallel changes to resx is needed.